packages/features/users/components/UserTable/UserListTable.tsx

```diff
@@ -169,7 +176,7 @@ export function UserListTable() {
       {
         id: "member",
         accessorFn: (data) => data.email,
-        header: `Member (${totalDBRowCount})`,
+        header: `Member (${isPlatformUser ? totalFetched : totalDBRowCount})`,
```

Comment:
We don't show the totalDBRowCount for platform since that would include managed users as well